.github/workflows/test.yml

```diff
@@ -27,7 +27,6 @@ jobs:
       - run: pnpm install
       - run: pnpm run lint
       - run: pnpm run test:types
-      - run: pnpm run -r dev:prepare
```

Comment:
I think this was right, it was more about also adding it to the root test script 😅

Reply:
yep! should be back 😁